sdk/core/azure-json/src/main/java/com/azure/json/package-info.java

```diff
@@ -2,6 +2,499 @@
 // Licensed under the MIT License.
 
 /**
- * Contains general classes for handling JSON.
+ * <p>The Azure JSON library provides interfaces for stream-style JSON reading and writing. Stream-style reading and
+ * writing has the type itself defines how to read JSON to create and instance of itself and how it writes JSON.
+ * Azure JSON also provides the ability to provide implementations for JSON reading and writing by offering a service
+ * provider interface to load implementations from the classpath, the library provides a default implementation if
+ * one cannot be found on the classpath.</p>
+ *
```

Comment:
Maybe it is worth mentioning here the reason why this library exists - to lessen our dependency on external libraries.

Reply:
I'll think about this, it seems a bit orthogonal to why the user would be reading this documentation. I feel is they're at this point they are already planning to use this and don't care too much about the Azure SDK dependencies situation.